phpBB/composer.json

```diff
@@ -48,7 +48,8 @@
 		"symfony/routing": "~3.1",
 		"symfony/twig-bridge": "~3.1",
 		"symfony/yaml": "~3.1",
-		"twig/twig": "^1.0"
+		"twig/twig": "^1.0",
+		"doctrine/dbal": "v2.5.12"
```

Comment:
yes but I don't think it i a good idea to use 2 different DBAL in the project. If we switch to Doctrine one day then it will be acceptable to have both during the transition. But here it is not the case and I don't think anything here requires doctrine